### PR TITLE
Add superset client to end to end tests

### PR DESCRIPTION
**Pull request by carlosms**
Part of #131.

This PR adds a client to use the superset API from the end to end tests.
For now it checks the list of dashboards, and SQL queries.

Next step would be to call the bblfsh-web endpoint on superset to test that it can parse files.

### CODE REVIEW

**Review by smacker — APPROVED** [review]

File: test/init_orgs_test.go [hunk 1]
```diff
@@ -118,4 +118,44 @@ github.com/golang-migrate/migrate
...
+
+	// Test metadata queries through superset
+	s.T().Run("superset-metadata", func(t *testing.T) {
+		req := require.New(t)
+
+		rows, err := client.metadata("select * from organizations")
+		req.NoError(err)
+		req.Len(rows, 1)
+
+		s.Equal("golang-migrate", rows[0]["login"])
```
Comment (dpordomingo) [inline test/init_orgs_test.go:159]: Sorry if already discussed and I missed it, but: can we rely on this org?

**Review by dpordomingo — APPROVED** [review]
LGTM

Nice idea using superset json api in the `supersetClient` to abstract the process in the test cases!

File: test/init_orgs_test.go (hunk 1 above)
Comment (carlosms) [reply test/init_orgs_test.go:159]: This was a conversation in the design doc between @smacker and me. The plan was to use golang-migrate for the initial milestone, and then later without rushing create a new one from scratch, with the minimal resources needed to test both sourced-ce and ghsync.
Maybe I should have opened an issue for this beforehand.

**Review by se7entyse7en — APPROVED** [review]
Cool! 👏

**Pull request merged**